addons/account/models/company_onboarding.py

```diff
@@ -0,0 +1,163 @@
+        self.env.user.company_id.onboarding_account_invoice_closed = True
+
+    @api.model
+    def onboarding_company(self, company_id):
+        """ Called by onboarding panel above the customer invoice list. """
+        action = self.env.ref('account.action_open_onboarding_company').read()[
+            0]
+        action['res_id'] = company_id
+        return action
+
+    @api.model
+    def onboarding_invoice(self, company_id):
+        """ Called by onboarding panel above the customer invoice list. """
+        action = self.env.ref('account.action_open_onboarding_invoice').read()[
+            0]
+        action['res_id'] = company_id
+        return action
+
+    def _get_sample_account(self, type_name, code_name):
+        """ If no account with code `code_name` exists, create a new one
+            with type `type_name` and code `code_name` and returns it """
+        sample_account = self.env['account.account'].search([('code', '=', code_name)], limit=1)
+
+        if len(sample_account) == 0:
+            sample_user_type_id = self.env['account.account.type'].search(
+                [('type', '=', type_name)], limit=1)
+            sample_account = self.env['account.account'].create({
+                'name': 'Sample Account ({})'.format(code_name),
+                'code': code_name,
+                'user_type_id': sample_user_type_id.id,
+                'reconcile': True,
+                'company_id': self.env.user.company_id.id,
+            })
+
+        return sample_account
+
+    def _get_sample_invoice(self):
+        """ Get a sample invoice or create it as well as a sample account,
+           journal, partner and line if it does not exist. """
+
+        sample_partner_name = 'Sample Partner'
+        sample_partner = self.env['res.partner'].search([('name', '=', sample_partner_name)], limit=1)
+
+        account_other_id = self._get_sample_account('other', 'SACOO').id
+        if len(sample_partner) == 0:
+            sample_partner = self.env['res.partner'].create({
+                'name': sample_partner_name,
+                'customer': True,
+                'email': self.env.user.email,
+                'property_account_payable_id': account_other_id,
+                'property_account_receivable_id': account_other_id,
+            })
+
+        sample_journal_code = 'SAJO'
+        sample_journal = self.env['account.journal'].search([('code', '=', sample_journal_code)], limit=1)
+        if len(sample_journal) == 0:
+            sample_journal = self.env['account.journal'].create({
+                'name': 'Sample Journal',
+                'code': sample_journal_code,
+                'type': 'sale',
+            })
+        sample_invoice_name = 'SAMPLE_INVOICE'
+        sample_invoice = self.env['account.invoice'].search([('name', '=', sample_invoice_name)], limit=1)
+        if len(sample_invoice) == 0:
+
+            sample_invoice = self.env['account.invoice'].create({
```

Comment:
Are we sure that it won't be possible to validate this invoice ? 

Are we sure that it won't generate accounting entries ? 

Are we sure that it won't generate followups ?